src/main/resources/avro/referencemethods.avdl

```diff
@@ -325,4 +386,99 @@ GetSequenceBasesResponse getSequenceBases(
     /** Additional request parameters to restrict the query. */
     GetSequenceBasesRequest request) throws GAException;
 
+/****************  /joins/search  *******************/
+/**
+This request maps to the body of `POST /joins/search` as JSON. Specifies a
+number of filters, all of which must be satisfird by each result returned.
+*/
+record SearchJoinsRequest {
+  /**
+  If present, return only `Join`s which belong to this reference set.
```

Comment:
Again, we should clarify that exactly one of referenceSetId or variantSetId must be non-null. We should also use the same language (we use 'if present' here, and 'if non-null above').